CONTRIBUTING.md

```diff
@@ -135,3 +135,12 @@ Doc: update Developer.md
 
 We enabled [stale bot](https://github.com/probot/stale) for house keeping. An
 Issue or Pull Request becomes stale if no any inactivity for 60 days.
+
+## For Mac and Windows user
+
+kepler currently focus on linux platform.
+for other platforms, to make kepler is easy for anyone contributes from 
+any platform, we are welcome any benefits(PRs) for kepler including 
```

Comment:
```suggestion
any platform, we are welcome any benefits(PRs) for kepler including
```
Sorry about the trailing whitespace.
Feel free to drop me from the co-authored by, it looks like conventional commits doesn't support it.